doc/sklearn.rst

```diff
@@ -22,6 +22,20 @@ scikit-learn application by using the following commandline flag::
 
     python -m daal4py my_application.py
 
+If using scikit-learn from intel conda channel, then 
```

Comment:
```suggestion
If you are using scikit-learn from Intel conda channel, then 
```

wouldn't `Intel(R) Distribution for Python` be a correct way to name "scikit-learn from Intel conda channel"?

Reply:
This is a good question, I will ask about it

Reply:
Fixed